myapp/templates/layout.html

```diff
@@ -17,32 +17,29 @@
     {% endblock %}
 
     <style>
-        #welcome_msg{
+        #welcomeMsg{
             font-size: 60px;
             text-align: center;
         }
-        .invisible, .temporary_message {
+        .invisible, .temporaryMessage {
             display: none;
         }
         /*Show a symbol in a sortable table's column headers to indicate that the table is sortable */
         table.sortable th:not(.sorttable_sorted):not(.sorttable_sorted_reverse):not(.sorttable_nosort):after { 
             content: " \25B4\25BE" 
         }
-        .tg  {border-collapse:collapse;border-spacing:0;}
-        .tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
-        .tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
-        .tg .tg-yw4l{vertical-align:top}
         #dLabel{color: grey;}
+         
     </style>
     <script>
-        function Generate_lab_id(lab_name,class_name,prof_name){
-            return lab_name+'_'+class_name+'_'+prof_name;
+        function generateLabId(labName,className,professorName){
```

Comment:
what linter are you using?

Also if your javascript grows in complexity, you're going to start wanting to rely on `<script src="path/to/my/static/js/stuff">`, which might make using a linter easier as well.